runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsConfig.java

```diff
@@ -28,18 +28,23 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.Range;
 
 @ConfigGroup("inventorytags")
 public interface InventoryTagsConfig extends Config
 {
 	enum DisplayMode
 	{
 		OUTLINE,
-		UNDERLINE
+		UNDERLINE,
+		OUTLINE_FILL
```

Comment:
Instead of adding another option here, I'd advise going the route of multiple toggles, [similar to the approach used in the NPC Indicators Plugin](https://github.com/runelite/runelite/blob/3c7a6380798f9821ec0477ee3746819f69e2897f/runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsConfig.java#L37-L78) so users can optionally enable outline, underline, and/or fill. (Maintaining the opacity option for the fill color is a good idea)

Reply:
I removed the DisplayMode enums and followed the approach shown in the NPC indicators for multiple toggles. I also lowered the default opacity to 50 because 100 felt little too opaque. Thanks.